### ISSUE TITLE
docker-compose up  ，Run failed

### ISSUE THREAD

**Issue by dai-chao** [dotenx/dotenx#329]
![f203fc3324d6294c9b2fabc25414d65](https://github.com/dotenx/dotenx/assets/45806795/1d324bb2-08a2-426c-be9f-a865291b61a4)

**Comment by mkamrani** [comment]
Thanks for reporting the issue. Let me look into this.

**Comment by mkamrani** [comment]
I'm gonna create a PR for this issue this week. I'm cleaning up the repository and some of the containers are already combined into a single task.

**Comment by dai-chao** [comment]
Thank you very much

**Comment by Hojjat-1** [comment]
I created a PR and it's had been merged now, in this PR I changed docker-compose file so you should be able to run `docker-compose up` without error. Please try again and let me know if you face with the error. Thanks for your support ❤️

**Comment by dai-chao** [comment]
Has it been merged into the main branch? I just tried pulling it and it still doesn't work

**Comment by Hojjat-1** [comment]
Yes my PR has been merged on main branch. Please make sure you pull the latest commits and send me a screenshot if the problem persists.
